src/site/paragraphs/lists_of_links.drupal.liquid

```diff
@@ -2,31 +2,100 @@
+
+        <ul class="usa-unstyled-list">
+          <!-- Links -->
+          {% for fieldLink in vaParagraph.entity.fieldLinks %}
+            <li class="vads-u-padding-y--1">
+              <a href="{{ fieldLink.url.path }}" rel="noreferrer noopener">
+                {{ fieldLink.title }}
+              </a>
+            </li>
+          {% endfor %}
+
+          <!-- See more articles link -->
+          {% if vaParagraph.entity.fieldLink.url.path %}
+            <li class="vads-u-padding-y--1">
+              <a class="vads-u-text-decoration--none" href="{{ vaParagraph.entity.fieldLink.url.path }}" rel="noreferrer noopener">
+                {{ vaParagraph.entity.fieldLink.title }} <i class="fa fa-chevron-right vads-u-padding-left--0p5 vads-u-font-size--sm" aria-hidden="true" role="presentation"></i>
+              </a>
+            </li>
+          {% endif %}
+        </ul>
+      </div>
+    {% endif %}
   {% endfor %}
+
+  <!-- Extra Categories -->
+  <div aria-hidden="true" class="vads-u-display--none vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row medium-screen:vads-u-flex-wrap--wrap" data-hidden-lists_of_links-entity-id="{{ entity.entityId }}">
+    {% for vaParagraph in formattedVaParagraphs %}
+      {% if forloop.index > 6 %}
+        <div class="vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-l-col--3 medium-screen:vads-u-margin-right--8">
+          <!-- Section header -->
+          <h3>{{ vaParagraph.entity.fieldSectionHeader }}</h3>
+
+          <ul class="usa-unstyled-list">
+            <!-- Links -->
+            {% for fieldLink in vaParagraph.entity.fieldLinks %}
+              <li class="vads-u-padding-y--1">
+                <a href="{{ fieldLink.url.path }}" rel="noreferrer noopener">
+                  {{ fieldLink.title }}
+                </a>
+              </li>
+            {% endfor %}
+
+            <!-- See more articles link -->
+            {% if vaParagraph.entity.fieldLink.url.path %}
+              <li class="vads-u-padding-y--1">
+                <a class="vads-u-text-decoration--none" href="{{ vaParagraph.entity.fieldLink.url.path }}" rel="noreferrer noopener">
+                  {{ vaParagraph.entity.fieldLink.title }} <i class="fa fa-chevron-right vads-u-padding-left--0p5 vads-u-font-size--sm" aria-hidden="true" role="presentation"></i>
+                </a>
+              </li>
+            {% endif %}
+          </ul>
+        </div>
+      {% endif %}
+    {% endfor %}
+  </div>
 </div>
+
+{% if formattedVaParagraphs.length > 6 %}
+  <script>
+    function onShowAllTopics() {
+      var extraParagraphsElement = document.querySelector('[data-hidden-lists_of_links-entity-id="{{ entity.entityId }}"]');
+
+      // Escape early if we are unable to find the hidden categories.
+      if (!extraParagraphsElement) {
+        console.warn('Unable to find hidden lists of links DOM nodes.');
+        return;
+      }
```

Comment:
I think we can scrap both of the blocks with the `console.warn`s and accept that if those elements aren't found, we get a runtime error. This function is only called on click, so the result would be the same either way (a broken toggle.)